src/pages/HH_ManageHousehold.page

```diff
@@ -1,180 +1,5 @@
-            <input type="text" class="gwField-input" id="hhName" name="hhName" />
-            <input type="checkbox" class="gwPreventUI" id="hhAutoName" name="hhAutoName" /><label for="hhAutoName">{!$label.npo02__AutoName}</label>
-        </div>
-        <div class="gwField">
-            <label for="hhFormalGreeting" class="gwField-label">{!$ObjectType.npo02__Household__c.Fields.npo02__Formal_Greeting__c.Label}</label>
-            <input type="text" class="gwField-input" id="hhFormalGreeting" name="hhFormalGreeting" />
-            <input type="checkbox" class="gwPreventUI" id="hhAutoFormalGreeting" name="hhAutoFormalGreeting" /><label for="hhAutoFormalGreeting">{!$label.npo02__AutoFormalGreeting}</label>
-        </div>
-        <div class="gwField">
-            <label for="hhInformalGreeting" class="gwField-label">{!$ObjectType.npo02__Household__c.Fields.npo02__Informal_Greeting__c.Label}</label>
-            <input type="text" class="gwField-input" id="hhInformalGreeting" name="hhInformalGreeting" />
-            <input type="checkbox" class="gwPreventUI" id="hhAutoInformalGreeting" name="hhAutoInformalGreeting" /><label for="hhAutoInformalGreeting">{!$label.npo02__AutoInformalGreeting}</label>
-        </div>
-    </div>
-</div>
-
-<apex:form >
-<!-- inputText so that the javascript can write back to the multiselect field -->
-<apex:inputText value="{!hhO.npo02__SYSTEM_CUSTOM_NAMING__c}" styleClass="gwHidden sfField sfField-SYSTEM_CUSTOM_NAMING__c" />
-<apex:inputField value="{!hhO.Name}" styleClass="gwHidden sfField sfField-Name" />
-<apex:inputField value="{!hhO.npo02__Formal_Greeting__c}" styleClass="gwHidden sfField sfField-Formal_Greeting__c" />
-<apex:inputField value="{!hhO.npo02__Informal_Greeting__c}" styleClass="gwHidden sfField sfField-Informal_Greeting__c" />
-<!-- Mailing Street on contacts is multi-line, but isn't in NPSP - using inputTextArea instead of inputField so that the CRLF is replaced by a space instead of deleted - still not perfect! -->
-<apex:inputTextarea value="{!hhO.npo02__MailingStreet__c}" styleClass="gwHidden sfField sfField-MailingStreet__c" />
-<apex:inputField value="{!hhO.npo02__MailingCity__c}" styleClass="gwHidden sfField sfField-MailingCity__c" />
-<apex:inputField value="{!hhO.npo02__MailingState__c}" styleClass="gwHidden sfField sfField-MailingState__c" />
-<apex:inputField value="{!hhO.npo02__MailingPostalCode__c}" styleClass="gwHidden sfField sfField-MailingPostalCode__c" />
-<apex:inputField value="{!hhO.npo02__MailingCountry__c}" styleClass="gwHidden sfField sfField-MailingCountry__c" />
-
-<apex:pageBlock id="hhDetails" rendered="{!hhO.id!=null}" title="{!$Label.npo02__HouseholdDetails}">
-
-<apex:pageBlockSection columns="2">
-
+<apex:page standardController="npo02__Household__c" extensions="HH_ManageHH_CTRL" title="{!$Label.npo02__ManageHousehold}" standardStylesheets="false" showHeader="true" docType="html-5.0"  >
+        
+	<apex:include pageName="HH_ManageHH"/>
```

Comment:
HH_ManageHH is the new visualforce page that wraps the lightning component.
HH_ManageHousehold is the old page for use with Household Objects.  It can now be just a wrapper to the shared HH_ManageHH page.
HH_ManageHHAccount is the old page for use with Household Accounts.  It will also become a wrapper over HH_ManageHH, but I am not ready to replace it yet since I still want to bring up the old ui for testing.